doc/compiler/jitaas/Usage.md

```diff
@@ -20,38 +20,38 @@ OpenJDK Assembly Exception [2].
 SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-JITaaS adds an additional two *personas* to the `java` executable: client mode and server mode. There are two new command line options that can be used to select a persona: `-XX:JITaaSServer` and `-XX:JITaaSClient`.
+JITaaS adds an additional two *personas* to the `java` executable: client mode and server mode. There are two new command line options that can be used to select a persona: `-XX:+StartAsJITServer` and `-XX:+UseJITServer`.
```

Comment:
I think we should delete the mention of JITaaS

Reply:
Will replace 
`JITaaS` => `JITServer` in all documents under `doc/compiler/jitaas` at where it applies. 
`doc/compiler/jitaas`=> `doc/compiler/jitserver`

Reply:
will leave the directory name change out from this PR for now for easier diff comparison.